test/integration/narrow/routes/endemics/check-answers.test.js

```diff
@@ -533,7 +534,7 @@ describe('Check answers test', () => {
         getEndemicsClaimMock.mockImplementation(() => {
           return {
             typeOfLivestock: 'pigs',
-            typeOfReview: 'review',
+            typeOfReview: 'R',
```

Comment:
How come this has changed?

Reply:
Its largely irrelevant, but typeOfReview is R or E - you can see in the other test similar to this one that it has typeOfReview E